src/LanguageServer/Impl/Disposables/DisposeToken.cs

```diff
@@ -1,82 +0,0 @@
-﻿// Python Tools for Visual Studio
-// Copyright(c) Microsoft Corporation
-// All rights reserved.
-//
-// Licensed under the Apache License, Version 2.0 (the License); you may not use
-// this file except in compliance with the License. You may obtain a copy of the
-// License at http://www.apache.org/licenses/LICENSE-2.0
-//
-// THIS CODE IS PROVIDED ON AN  *AS IS* BASIS, WITHOUT WARRANTIES OR CONDITIONS
-// OF ANY KIND, EITHER EXPRESS OR IMPLIED, INCLUDING WITHOUT LIMITATION ANY
-// IMPLIED WARRANTIES OR CONDITIONS OF TITLE, FITNESS FOR A PARTICULAR PURPOSE,
-// MERCHANTABLITY OR NON-INFRINGEMENT.
-//
-// See the Apache Version 2.0 License for specific language governing
-// permissions and limitations under the License.
-
-using System;
-using System.Threading;
-using static System.FormattableString;
-
-namespace Microsoft.DsTools.Core.Disposables {
-    public sealed class DisposeToken {
```

Comment:
Duplicate?

Reply:
It was in LS, but engine could not use it, I moved it to infra. We soon will need some DSTools.Core :-)

Reply:
Speaking of which, I can port disposable tests from RTVS too.